app.module.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from '@nestjs/common';
+import { ConfigModule } from '@nestjs/config';
+
+@Module({
+    imports: [ConfigModule.forRoot({
+        isGlobal: true,
+    })],
+})
+
+export class AppModule {}
```

Comment:
What does this file do? I've seen `app.module.ts` inside `api/"api name"/src/app`, but it looks this file exists in the root directory.